docs/content/migration.md

```diff
@@ -0,0 +1,545 @@
+| fab-speed-dial     |   [Docs][78] |   -          | Not implemented                                        |
+| fab-toolbar        |   [Docs][79] |   -          | Not implemented                                        |
+| grid-list          |   [Docs][50] |   [Docs][9]  |                                                        |
+| highlight-text     |   [Docs][80] |   -          | Not implemented                                        |
+| icon               |   [Docs][51] |   [Docs][10] |                                                        |
+| ink-ripple         |   [Docs][81] |   [Docs][19] | `matRipple`                                            |
+| input              |   [Docs][52] |   [Docs][5]  | `matInput`                                             |
+| input-container    |   [Docs][82] |   [Docs][83] | `mat-form-field`                                       |
+| list               |   [Docs][53] |   [Docs][8]  |                                                        |
+| menu               |   [Docs][54] |   [Docs][17] |                                                        |
+| menu-bar           |   [Docs][84] |   -          | Not implemented                                        |
+| nav-bar            |   [Docs][85] |   [Docs][86] | `mat-tab-nav-bar`                                      |
+| nav-item           |   [Docs][87] |   [Docs][88] | `mat-tab-link`                                         |
+| optgroup           |   [Docs][89] |   [Docs][90] |                                                        |
+| option             |   [Docs][91] |   [Docs][23] |                                                        |
+| progress-linear    |   [Docs][55] |   [Docs][12] | `mat-progress-bar`                                     |
+| progress-circular  |   [Docs][56] |   [Docs][11] | `mat-progress-spinner`                                 |
+| radio-button       |   [Docs][57] |   [Docs][4]  |                                                        |
+| radio-group        |   [Docs][92] |   [Docs][93] |                                                        |
+| select             |   [Docs][59] |   [Docs][23] |                                                        |
+| select-on-focus    |   [Docs][94] |   -          | Not implemented                                        |
+| sidenav            |   [Docs][60] |   [Docs][6]  |                                                        |
+| sidenav-focus      |   [Docs][95] |   -          | [autofocus][96] only supports focusing the first focusable element.|
+| switch             |   [Docs][61] |   [Docs][14] | `mat-slide-toggle`                                     |
+| slider             |   [Docs][62] |   [Docs][16] |                                                        |
+| slider-container   |   [Docs][97] |   -          | Not implemented                                        |
```

Comment:
I kind of want to say here "Replaced by literally putting whatever you want next to mat-slider"

Reply:
Right, similar to https://github.com/angular/material/pull/11690#discussion_r274240311, there is no matching API, but we could add a section to explain how to accomplish some of these use cases in Angular Material? 

Certainly the answer for a number of these is that such an API is not needed because there is a better native or Angular-specific way to accomplish the same thing.